uber_sante/__init__.py

```diff
@@ -0,0 +1,13 @@
+import os
+from flask import Flask
+from config import Config
+from uber_sante.utils.dbutil import DBUtil
+
+app = Flask('uber_sante')
+app.config.from_object(__name__)
+
+Config.DATABASE = os.path.join(app.root_path + '/db/', 'database.db')
```

Comment:
This line is already in the Config class, you can remove it altogether

Reply:
Good find